src/ray/gcs/tables.cc

```diff
@@ -86,7 +96,12 @@ Status Log<ID, Data>::AppendAt(const JobID &job_id, const ID &id,
 template <typename ID, typename Data>
 Status Log<ID, Data>::Lookup(const JobID &job_id, const ID &id, const Callback &lookup) {
   num_lookups_++;
-  auto callback = [this, id, lookup](const std::string &data) {
+  const auto start_time = current_time_ms();
+  auto callback = [this, id, lookup, start_time](const std::string &data) {
+    const auto end_time = current_time_ms();
+    stats::RedisLatency().Record(
+        end_time - start_time,
+        {{stats::NodeAddressKey, RayConfig::instance().node_address()}});
```

Comment:
The same code is repeated. I think we can avoid this by moving the measuring to `RunAsync`